geonode/resource/manager.py

```diff
@@ -444,6 +445,7 @@ def copy(self, instance: ResourceBase, /, uuid: str = None, owner: settings.AUTH
                 _resource = copy.copy(instance.get_real_instance())
                 _resource.pk = _resource.id = None
                 _resource.uuid = uuid or str(uuid1())
+                _resource.date = now()
```

Comment:
`date` is not creation date, it refers to the resource contents, so it shouldn't be updated. Cretion date is `created` and update date is `last_updated`, and they are automatically set to now.